File: 1-two-sum/two-sum.py
class Solution(object):
    def twoSum(self, nums, target):
        n =len(nums)
        for i in range(n): 
            q = target-nums[i]
            for j in range(i+1,n):
                if q == nums[j]:
                    return[i,j]
                    

File: 1-two-sum/two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        int i, j, n = nums.size(), q = 0;
        
        for (i = 0; i < n; i++) {
            q = target - nums[i];
            
            for (j = i + 1; j < n; j++) {
                // Check against nums[j], not nums[i]
                if (q == nums[j]) {
                    return {i, j};
                }
            }
        }
        
        return {}; // Fallback return to satisfy the compiler
    }
};